ucm2/HDA-Intel/Hdmi.conf

```diff
@@ -0,0 +1,79 @@
+# Use case Configuration for HDA-Intel
+
+If.hdmi1 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=3 Jack'"
+	}
+	True {
+		SectionDevice."HDMI1" {
+			Comment "HDMI1/DP1 Output"
+
+			EnableSequence [
+				cset "name='IEC958 Playback Switch' on"
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch' off"
+			]
+
+			Value {
+				PlaybackPriority 500
+				PlaybackPCM "hw:${CardId},3"
+				JackControl "HDMI/DP,pcm=3 Jack"
+			}
+		}
+	}
+}
+
+If.hdmi2 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=4 Jack'"
+	}
+	True {
+		SectionDevice."HDMI2" {
+			Comment "HDMI2/DP2 Output"
+
+			EnableSequence [
+				cset "name='IEC958 Playback Switch',index=1 on"
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch',index=1 off"
+			]
+
+			Value {
+				PlaybackPriority 600
+				PlaybackPCM "hw:${CardId},4"
+				JackControl "HDMI/DP,pcm=4 Jack"
```

Comment:
Change the 4 to 7
				PlaybackPCM "hw:${CardId},7"
				JackControl "HDMI/DP,pcm=7 Jack"